doc/source/bootstrap.rst

```diff
@@ -14,19 +14,17 @@ If the :variable:`wsrep_cluster_address` variable is empty,
 |PXC| assumes that this is the first node and initializes the cluster.
 
 Instead of changing the configuration,
-start the first node using the following command:
+start the first node use the following command:
```

Comment:
I would revert "using" here. Or split this sentence in two: "Instead of changing the configuration, start the first node. Use the following command:"

Reply:
Fixed.